cockpit/src/subscriptions-view.jsx

```diff
@@ -71,11 +71,38 @@ class InstalledProducts extends React.Component {
 
     render() {
         let columnTitles = [_("Product name")];
+
+        let sca_mode;
+
+        sca_mode = false;
+        if (this.props.org) {
+            if ("contentAccessMode" in this.props.org) {
+                if (this.props.org.contentAccessMode === "org_environment") {
+                    sca_mode = true;
+                }
+            }
+        }
+
+        let card_actions;
+        if (sca_mode === false) {
+            card_actions = (
+                <CardActions>
+                    <Button
+                            isDisabled={this.state.attaching_in_progress ||this.props.status === 'unknown'}
```

Comment:
It seems that this project is missing a linter :) Would be nice to have a space around operators, but feel free to ignore and we can add a linter and fix this later on.

Reply:
We use `webpack`, which complains about some stylish errors. PyCharm has also some buil-in linter, but this was just ignored by all of them. We welcome any suggestions how to improve it. Fixed.